NGINX Plus Release 34 (#343)
nginx/documentation: content/nginx/deployment-guides/single-sign-on/oidc-njs/active-directory-federation-services.md

---

> description: Enable OpenID Connect-based single-sign for applications proxied by NGINX
>   Plus, using Microsoft AD FS as the identity provider (IdP).
> docs: DOCS-463
> doctypes:
> - task
> title: Single Sign-On with Microsoft AD FS and njs
> toc: false
> weight: 100

Reviewer: ```suggestion
description: Enable OpenID Connect-based single-sign for applications proxied by NGINX
  Plus, using Microsoft AD FS as the identity provider (IdP).
docs: DOCS-463
type:
- how-to
title: Single Sign-On with Microsoft AD FS and njs
toc: false
weight: 100
product: NGINX-PLUS
```
